recipients-resolver/src/main/java/com/redhat/cloud/notifications/recipients/resolver/rbac/RbacServiceToService.java

```diff
@@ -40,6 +40,7 @@ RbacGroup getGroup(
     Page<RbacUser> getGroupUsers(
             @HeaderParam("x-rh-rbac-org-id") String orgId,
             @PathParam("groupId") UUID groupId,
+            @QueryParam("admin_only") Boolean adminOnly,
```

Comment:
They made the field a `string` in RBAC so I'm not sure we can use a `Boolean` here.

Related question: https://github.com/RedHatInsights/insights-rbac/pull/857#discussion_r1950455502

Reply:
Nevermind, they changed it to `boolean` after the initial commit.